ludwig/utils/backward_compatibility.py

```diff
@@ -81,20 +81,17 @@ def wrap(fn: Callable[[Dict], Dict]):
 
 def upgrade_to_latest_version(config: Dict):
     """Updates config from an older version of Ludwig to the current version. If config does not have a
-    "ludwig_version" key, no updates are applied.
+    "ludwig_version" key, all updates are applied.
```

Comment:
In line 1361 of api.py LudwigModel.load() I added a similar condition: if no version in config, assume an old version.  At this time I was thinking we should upgrade on load but error on init for old configs.  On second thought, I agree this is better... and with this change the condition in api.py load() should be removed.

Reply:
Thanks! Removed code at line 1361.